std/bigint.d

```diff
@@ -578,7 +582,13 @@ string toHex(const(BigInt) x)
     return outbuff;
 }
 
-// Returns the absolute value of x converted to the corresponding unsigned type
+/** Returns the absolute value of x converted to the corresponding unsigned
```

Comment:
Hmm, I had in mind replacing the above line with the `Returns:` line, but meh, it's getting too nitpicky. LGTM.